refs/x-compile.sh

```diff
@@ -1671,8 +1673,14 @@ fi
 echo "* Building OpenSSH . . ."
 echo ""
 cd ..
-tar -I pigz -xvf /usr/portage/distfiles/openssh-7.8p1.tar.gz
-cd openssh-7.8p1
+# NOTE: Because of course, everything is terrible, OpenSSH >= 7.9 stopped supporting OpenSSL 0.9.8...
```

Comment:
Ummm what? >_>

Reply:
Yeeep.

It doesn't even build against 1.0.2 without a patch to bypass the preprocessor check.... -_-".